rolling/ci-nightly-connext.yaml

```diff
@@ -26,7 +26,7 @@ jenkins_job_priority: 50
 jenkins_job_schedule: 15 23 * * *
 jenkins_job_timeout: 300
 jenkins_job_weight: 4
-package_selection_args: '--packages-ignore fastcdr foonathan_memory_vendor rosbag2_converter_default_plugins --packages-ignore-regex .*cyclonedds.* .*fastrtps.*'
+package_selection_args: '--packages-ignore foonathan_memory_vendor rosbag2_converter_default_plugins --packages-ignore-regex .*cyclonedds.* .*fastrtps.*'
```

Comment:
Doesn't `rmw_connextdds` require the rosidl packages for Fast-RTPS as well?

Reply:
Oh, I missed that in the regex.  Good call, I'll remove that as well.